src/jit_no_ir.h

```diff
@@ -222,6 +222,24 @@ class EmitX64 {
         reinterpret_cast<EmitX64*>(this_ptr)->DoInterruptsAndRunDebug();
     }
 
+    FORCE_INLINE void LookupBlock() {
+        auto& vec = block_cache[regs.pc];
+        for (auto& [key, block] : vec) {
+            if (key == blk_key) {
+                current_blk = &block;
+                return;
+            }
+        }
+
+Compile:
```

Comment:
was there a goto here 😭

Reply:
You saw nothing 😶‍🌫️

Reply:
[![image](https://github.com/GPUCode/teakra/assets/44909372/9740168f-dd86-4763-beb8-fbbbe610d7b1)
](https://media1.tenor.com/m/EWqQ_gHKaMMAAAAd/sleepy-tired.gif)